### PR TITLE
CP-38530: Remove PuTTY

### PR DESCRIPTION
**Pull request by danilo-delbusso**
To be merged together with https://github.com/xenserver/xenadmin/pull/2919 as part of CP-38530.

### CODE REVIEW

**Review by lindig — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -12,8 +12,6 @@ libraries:
   and clients in the .NET environment;
 * log4net (v2.0.12) - a library providing logging services for purposes
   of application debugging and auditing;
```
Comment (kc284) [inline README.md:14]: Remove also `toolset v142 for C++` further down in the README, it's not needed for anything else.

File: build.ps1 [hunk 2]
```diff
@@ -222,14 +222,6 @@ Get-ChildItem $PATCHES | where { $_.Name.StartsWith("patch-discutils") } |`
 'dll', 'pdb' | % { "$SCRATCH_DIR\DiscUtils\src\bin\Release\DiscUtils." + $_ } |`
   Move-Item -Destination $OUTPUT_48_DIR
 
-#prepare PuTTY
-
-mkdirClean "$SCRATCH_DIR\PuTTY"
-Expand-Archive -DestinationPath "$SCRATCH_DIR\PuTTY" -Path "$REPO\PuTTY\putty-src.zip"
-'version.h', 'licence.h' | % { "$SCRATCH_DIR\PuTTY\" + $_ } | Copy-Item -Destination "$SCRATCH_DIR\PuTTY\windows\"
-& $msbuild $SWITCHES $VS2019_CPP "$SCRATCH_DIR\PuTTY\windows\VS2012"
```
Comment (kc284) [inline build.ps1:230]: Remove also the definition of `$VS2019_CPP` in line 75.

**Review by kc284 — CHANGES_REQUESTED** [review]

**Comment by kc284** [comment]
Is there a reason why this is still a draft PR?

File: README.md [hunk 3]
```diff
@@ -35,10 +35,8 @@ directories. Please see the [LICENSE](LICENSE) file for more information.
 
 1. PowerShell 3.0 or above
 2. Net Framework 4.5 and 4.8 installed.
-3. Visual Studio build tools for 2019 (toolsversion 16.0 and toolset v142 for C++).
```
Comment (kc284) [inline README.md:40]: We still need VS 2019 toolsversion 16.

Comment (kc284) [reply README.md:40]: And the note about msbuild.

Comment (kc284) [reply README.md:40]: So basically only the toolset v142 needed removal.

Comment (danilo-delbusso) [reply README.md:40]: Ah sorry I had misunderstood. Will fix it shortly

**Review by kc284 — APPROVED** [review]

**Pull request merged**